File: sandbox/java/src/main/java/com/dropbox/sign_sandbox/TemplateDeleteDefaultExample.java
package com.dropbox.sign_sandbox;

import com.dropbox.sign.ApiException;
import com.dropbox.sign.Configuration;
import com.dropbox.sign.api.*;
import com.dropbox.sign.auth.*;
import com.dropbox.sign.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class TemplateDeleteDefaultExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        try
        {
            new TemplateApi(config).templateDelete(
                "f57db65d3f933b5316d398057a36176831451a35"
            );
        } catch (ApiException e) {
            System.err.println("Exception when calling Template#templateDelete");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: sandbox/dotnet/src/Dropbox.SignSandbox/TemplateDeleteDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TemplateDeleteDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        try
        {
            new TemplateApi(config).TemplateDelete(
                templateId: "f57db65d3f933b5316d398057a36176831451a35"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Template#TemplateDelete: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
